core/shared/src/test/scala-3.x/monocle/focus/FocusFieldSelectTest.scala

```diff
@@ -20,6 +20,11 @@ final class FocusFieldSelectTest extends munit.FunSuite {
   case class ConstraintBox[A <: AnyVal](a: A)
   case class Varargs[A](a: A*)
 
+  object Public {
+    case class Private(private[Public] a: Int)
+    def lens: Lens[Private, Int] = Focus[Private](_.a)
```

Comment:
out of curiosity, what happens if you try to create this lens outside of `Public`?